include/zephyr/drivers/led.h

```diff
@@ -46,6 +46,21 @@ struct led_info {
 	const uint8_t *color_mapping;
 };
 
+/**
+ * @brief Container for an LED information specified in devicetree.
+ *
+ * This type contains a pointer to and LED device and an LED index.
+ *
+ * @see LED_DT_SPEC_GET
+ * @see LED_DT_SPEC_GET_OR
+ */
+struct led_dt_spec {
+	/** LED device instance. */
+	const struct device *dev;
+	/** Index of the LED on the controller. */
+	uint32_t index;
+};
+
```

Comment:
For the readability of the `led.h` header it might be interesting to move this structure definition, just before the definitions of the `led_x_dt()` functions. So everything DT_SPEC related would be grouped in the same code block. And this block could be advertised with a nice comment:
```
/*
 * LED DT helpers.
 * /
```
What do you think ?

Reply:
Yeah sounds good, moved that block and added the comment, it should all be together now.